src/views/Item/ItemMeasurements.vue

```diff
@@ -16,32 +17,32 @@
       </template>
     </template>
 
-    <template v-if="allKpis.length" #default>
+    <template #default>
       <kpi-details v-if="showKpiDetails && activeKpi" :kpi="activeKpi" />
     </template>
-
-    <template v-else #default>
-      <empty-state
-        :icon="'exclamation'"
-        :heading="$t('empty.noKPIs.heading')"
-        :body="$t('empty.noKPIs.body')"
-      >
-        <router-link
-          v-if="hasEditRights"
-          :to="{ name: 'ItemAdmin', query: { tab: 'kpi' } }"
-        >
-          {{ $t('empty.noKPIs.linkText') }}
-        </router-link>
-      </empty-state>
-    </template>
   </page-layout>
+
+  <empty-page
+    v-else
+    :heading="$t('empty.noKPIs.heading')"
+    :body="$t('empty.noKPIs.body')"
+  >
+    <div v-if="hasEditRights" data-mode="dark">
+      <pkt-button
```

Comment:
Men hm, litt synd å bytte ut lenkene med knapper hvis det bare er for det stilistiske? Semantisk er de jo mer lenker enn knapper, og nå mister vi naise ting som å kunne høyre- og midtenklikke på dem. :/ Vil tro at det kanskje også er mindre UU-vennlig siden URL-en graves inn i en event handler?

Reply:
Ser ut som Punkt er enig:

> Husk, en tommelfingerregel er å skille mellom funksjonen til lenker og knapper:
>
>    Bruk lenker når brukeren skal komme til en ny side
>    Bruk knapper når brukerne skal utføre en konkret handling: logg inn, send, last opp osv.

Kunne vi brukt link card i stedet kanskje, hvis det skal være en lenke som er litt fremtredende og som skal se pen ut? Eller om vi gjerne vil beholde knappestilen, slenge noen av de Punkt button-klassene på et `<a>`-element?

Reply:
Ja, det var sant, tenkte ikke på det.